### ISSUE TITLE
Question: Support for Svelte / sapper?

### ISSUE THREAD

**Issue by georks** [nathancahill/split#458]
Hi,
this utility is exactly what I'm looking for and I would like to use it with sapper / svelte (https://sapper.svelte.dev/)
I have tried but did not get it running with sapper. Could you be of help?

**Comment by nathancahill** [comment]
Yes. You can use it with `onMount`. I'm also considering adding Svelte components. Stay tuned!
